src/app/gateways/components/RuleMatch.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <dl class="stack-small">
+    <template v-if="props.match.method">
+      <div class="match">
+        <dt>
+          <KBadge appearance="neutral">
+            Method
+          </KBadge>
+        </dt>
+
+        <dd>
+          <KBadge>{{ props.match.method }}</KBadge>
+        </dd>
+      </div>
+    </template>
+
+    <template v-if="props.match.path">
+      <div class="match">
+        <dt>
+          <KBadge appearance="neutral">
+            Path
+          </KBadge>
+        </dt>
+
+        <dd>
+          <span><span class="text-neutral">{{ t('http.api.property.type') }}:</span> {{ props.match.path.type }}</span>
+          <code>{{ props.match.path.value }}</code>
+        </dd>
+      </div>
+    </template>
+
+    <template v-if="props.match.queryParams && props.match.queryParams.length > 0">
+      <div
+        v-for="(param, index) in props.match.queryParams"
+        :key="index"
+      >
+        <div class="match">
+          <dt>
+            <KBadge appearance="neutral">
+              Query parameter
+            </KBadge>
+          </dt>
+
+          <dd>
+            <div class="list">
+              <span><span class="text-neutral">{{ t('http.api.property.type') }}:</span> {{ param.type }}</span>
+              <span>{{ param.name }}:{{ param.value }}</span>
+            </div>
+          </dd>
+        </div>
+      </div>
+    </template>
+
+    <template v-if="props.match.headers && props.match.headers.length > 0">
+      <div
+        v-for="(header, index) in props.match.headers"
+        :key="index"
+        class="match"
+      >
+        <dt>
+          <KBadge appearance="neutral">
+            Header
+          </KBadge>
+        </dt>
+
+        <dd>
+          <div class="list">
+            <span><span class="text-neutral">{{ t('http.api.property.type') }}:</span> {{ header.type ?? 'Exact' }}</span>
```

Comment:
I wanted to do this, but held off for the time being because the source of data is nested _quite_ deep in the _rules API. Eventually, we should probably do this though.

Reply:
Cool lets consider this in later PRs then 👍